### ISSUE TITLE
Email not rendering images

### ISSUE THREAD

**Issue by smilansky** [meteorhacks/meteor-ssr#36]
Hey I am attempting to send images and they are not rendering. I put the images in ./public/images/ and tried two methods of rendering them:

1. The conventional way:

```html
<p><img src="/images/Ecquire-Capturing-Data-from-Professional-Network.png"></p> 
```

This doesnt serve the image.

2. I created a template helper that returns Meteor.absoluteUrl. 
```html
<p><img src={{absoluteURL("/images/Ecquire-Capturing-Data-from-Professional-Network.png")}}></p> 

```

This errors with:

```
Exception in callback of async function: Error: Expected IDENTIFIER
I20151008-12:29:51.393(-7)? ...mg src={{absoluteURL("/images/Ecquire-Cap...
```

Any ideas? Seems like a simple syntax error. The only other option I can think of is to attach the absolute Image URL with a variable to the template data and then plug that in.

**Comment by bsbechtel** [comment]
I'm having the same issue. Did you end up having any luck figuring this out?

**Comment by smilansky** [comment]
Yes, you put url of where the image is hosted -       `<img src="http://[your-url-here]/images/Services.png" style="width:auto; height:auto; max-width:400px; max-height:400px;"></td>`

**Comment by chrise86** [comment]
So you have to manually specify the url? Is that the only way?

**Comment by marcinwal** [comment]
I am having the same problem; maybe you could help me out ? http://stackoverflow.com/questions/36498601/rendering-pictures-in-emails-template-in-meteor

**Comment by quangnd** [comment]
With HTML Email Templates, you MUST USE absolute path for image. So, please upload your image to your hosting and get link to display
